tests/test_examples/test_gsd.py

```diff
@@ -3,18 +3,28 @@
 
 def test_gsd_iluz(snapshot):
     from examples.gsd._01_gsd_iluz import (
-        categorized_intervals,
         long_trial_output,
         long_trial_output_modified,
-        matched_metrics_dict,
-        matches,
         short_trial_output,
-        unmatched_metrics_dict,
     )
 
     snapshot.assert_match(long_trial_output.gs_list_, "long_trial_output")
     snapshot.assert_match(long_trial_output_modified.gs_list_, "long_trial_output_modified")
     snapshot.assert_match(short_trial_output.gs_list_, "short_trial_output")
+
+
+def test_gsd_evaluation(snapshot):
+    from examples.gsd._02_gsd_evaluation import (
+        categorized_intervals,
+        detected_gsd_list,
+        matched_metrics_dict,
+        matches,
+        reference_gsd_list,
+        unmatched_metrics_dict,
+    )
+
+    snapshot.assert_match(reference_gsd_list, "reference_gsd_list")
```

Comment:
Is there a reason why you snapshotted the "input data". In general I would try to be conservative with the created snapshots. Less things to update in case of actual changes.